version.sh

```diff
@@ -1,2 +1,2 @@
-export ARTIFACT_NODE_VERSION="6.10.2"
-export START_NODE_VERSION="${ARTIFACT_NODE_VERSION}"
\ No newline at end of file
+export ARTIFACT_NODE_VERSION="10.15.3"
```

Comment:
The node version upgrade is out of scope for this PR
```suggestion
export ARTIFACT_NODE_VERSION="10.14.2"
```